node-services/cards-external-diffusion/src/domain/client-side/configService.ts

```diff
@@ -33,6 +33,7 @@ export default class ConfigService {
                 this.config = new ConfigDTO();
 
                 this.config.mailFrom = defaultConfig.mailFrom;
+                this.config.dailyEmailTitle = defaultConfig.dailyEmailTitle;
```

Comment:
Add the other two parameters : hourToSendDailyEmail and public minuteToSendDailyEmail